Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkTransformationManager.java

```diff
@@ -209,4 +209,138 @@ public ITransformedNetwork getSearchTransformedNetwork(String projectId, String
 
         return networkList;
     }
+    
+    
+    @Override
+    public ITransformedNetwork getSearchTransformedNetwork(List<String> projectIds, String conceptId)
+        throws QuadrigaStorageException {
+    	// get the transformed network and search for the concept id
+        List<INetwork> networkList = new ArrayList<>();
+        
+        for(String projectId : projectIds){
+        	List<INetwork> network = getNetworkList(projectId);
+        	if(network != null){
+        		networkList.addAll(network);
+        	}
+        }
+        
+        if (networkList.size()==0) {
+            return null;
+        }
+
+        // get the transformed network of all the networks in a project
+        ITransformedNetwork transformedNetwork = getTransformedNetworkOfAllProjects(networkList);
+
+        // Filter the nodes with the concept id
+        // add all the statement ids to a set
+        Set<String> statementIdSearchSet = new HashSet<String>();
```

Comment:
Is the following code basically the same as in the other method? Don't do that! Remember the DRY principal (don't repeat yourself). Find a better way (e.g. extract that into a method).